eureka2-core/src/main/java/com/netflix/eureka2/registry/SourcedEurekaRegistry.java

```diff
@@ -48,16 +48,10 @@
     Observable<ChangeNotification<T>> forInterest(Interest<T> interest, Source.Matcher sourceMatcher);
 
     /**
-     * Evict all registry info with the given source
+     * Evict all registry info for all sources except the single specified source
      * @return an observable of long denoting the number of holder items touched for the eviction
      */
-    Observable<Long> evictAll(Source source);
-
-    /**
-     * Evict all registry info for all sources
-     * @return an observable of long denoting the number of holder items touched for the eviction
-     */
-    Observable<Long> evictAll();
+    Observable<Long> evictAllExcept(Source source);
```

Comment:
How do we evict now replication channels? We need to put on the eviction queue a specific source.